autoscale/src/main/java/com/sequenceiq/periscope/repository/SubscriptionRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.periscope.repository;
+
+import java.util.List;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import com.sequenceiq.periscope.domain.Subscription;
+
+
+public interface SubscriptionRepository extends CrudRepository<Subscription, Long> {
+
+    List<Subscription> findByClientIdAndEndpoint(@Param("clientId") String clientId, @Param("endpoint") String endpoint);
```

Comment:
Named query could be replace with built in repository query generator, like findAllByClientIdAndEndpoint()